recipes/psyinf-gmtl/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
   "0.7.1":
     url: "https://github.com/psyinf/gmtl/archive/refs/tags/0.7.1.tar.gz"
-    sha256: "64e36b8c41b1829933921cd5a2f2825111840010b6d0e3aaa82c023c8fd7ebd5"
+    sha256: "e7fbc98d8714251655c1f2ab60d35500ca6c1c57e4887419c99ffb4a49fc5030"
```

Comment:
@psyinf  Why does the SHA-256 changed? Please, avoid it, it's considered a bad practice, prefer releasing a new version instead.

Reply:
@uilianries You're right. The reason are multiple changes of the CMakeList to get it to a conforming state. How to handle multiple changes here? I mean it is out of the question to change the version for every small change here, since it isn't even related to the code itself. Is there a way to get all the changes reviewable and have the version bump only just before the PR? However you are right, it most definitely is bad practice to change the referenced commit in while keeping the version the same